kmippp/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 add_library(
   kmippp
   STATIC
+  core_error.cpp
```

Comment:
Header files should be added to `add_library()` as well (it helps CMake to build Makefiles with proper dependencies). So, please add `core_error.hpp` here as well.

Reply:
fixed